src/main/python/main.py

```diff
@@ -1,12 +1,93 @@
+import sys
+from PyQt5.QtWidgets import (
+    QMainWindow, QMenu, QWidget, QTabWidget,
+    QTableView, QGraphicsView,
+    QMenuBar,
+    QVBoxLayout, QHBoxLayout,
+)
+from pyqtgraph import GraphicsLayoutWidget
+
 from fbs_runtime.application_context.PyQt5 import ApplicationContext
-from PyQt5.QtWidgets import QMainWindow
 
-import sys
+
+class SweepPage(QTableView):
+    def __init__(self):
+        super().__init__()
+
+
+class CellPage(QWidget):
+    def __init__(self):
+        super().__init__()
+
+        cell_features_table = QTableView()
+        cell_plots = CellPlotsView()
+
+        layout = QVBoxLayout()
+        self.setLayout(layout)
+
+        layout.addWidget(cell_features_table,1)
+        layout.addWidget(cell_plots,3)
+
+
+class CellPlotsView(QGraphicsView):
+    def __init__(self):
+        super().__init__()
+
+
+class MainWindow(QMainWindow):
+
+    def __init__(self):
```

Comment:
placing stuff in `__init__` is not unreasonable, but it can impede testing. This is because the class cannot be constructed without doing a bunch of work that is almost all extraneous to the purpose of the unit test. I don't think it's so bad with forward-facing UI classes like this (which are hard to unit test anyways), but as a general note it might be better to leave `__init__` simple and make an `init_ui` method that does the initialization.

Reply:
good point!